test/blocks/homepage-brick/homepage-brick.test.js

```diff
@@ -0,0 +1,77 @@
+import { readFile, sendMouse } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import { setLibs } from '../../../homepage/scripts/utils.js';
+
+setLibs('/libs');
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const { default: init } = await import('../../../homepage/blocks/homepage-brick/homepage-brick.js');
+
+function getMiddleOfElement(element) {
+  const { x, y, width, height } = element.getBoundingClientRect();
+
+  return {
+    x: Math.floor(x + window.pageXOffset + width / 2),
+    y: Math.floor(y + window.pageYOffset + height / 2),
+  };
+}
+
+describe('homepage-brick block', () => {
+  const blocks = document.querySelectorAll('.homepage-brick');
+  
+  describe('click variant', () => {
+    it('click variation', async () => {
+      const block = blocks[0];
+      await init(block);
+      expect(block.className).to.equal('homepage-brick click');
+      const directChildren = block.querySelectorAll(':scope > div');
+      expect(directChildren.length).to.equal(3);
+      expect(directChildren[0].className).to.equal('background first-background');
+      expect(directChildren[1].className).to.equal('background');
+      expect(directChildren[2].className).to.equal('foreground');
+      const h3 = block.querySelector('h3');
+      expect(h3).to.be.exist;
+      const link = block.querySelector('div.click-link');
+      expect(link).to.be.exist;
+      const { x, y } = getMiddleOfElement(block);
+      await sendMouse({ type: 'click', position: [x, y] });
```

Comment:
I know you will revisit this again, but wondering what will be the expect result of click event.

Reply:
clicking anywhere in the pod should take the user to the destination of the link.  So the href of the link is moved to a data attribute on the foreground element, and a click event makes the element behave like a link.  I had trouble getting the analytics and cosmetic of the block to look the way I wanted if I actually made the foreground a link.  But maybe we can revisit that.